### DISCUSSION TITLE
XPS conversion PDF lost curve filling

### DISCUSSION THREAD

**Discussion by gpjupc888** [pymupdf/PyMuPDF#1170] [Upstream Bugs]
_**Please provide all mandatory information!**_

## Describe the bug (mandatory)
When I converted the XPS file to PDF, the curve fill was lost in the PDF file.

## To Reproduce (mandatory)

import fitz
from tkinter import filedialog
path = filedialog.askopenfilename()
xps_doc = fitz.open(path)
pdfBytes = xps_doc.convert_to_pdf()
pdf_doc = fitz.open('pdf', pdfBytes)
save_path = path.split('.')[0] + '.pdf'
pdf_doc.save(save_path)

## Expected behavior (optional)
I hope it can be completely converted into A PDF file. Could you please help me find out where the problem is

## Screenshots (optional)
![image](https://user-images.githubusercontent.com/87957612/126925060-6c65d009-acf1-4f55-94b9-3a9d947a1834.png)
In Figure 1, the fourth plot is filled with color.
![image](https://user-images.githubusercontent.com/87957612/126925109-32a1b096-6f91-48d0-8229-e552e499e366.png)
But in Figure 2, which is my converted PDF file, the color fill of the fourth chart disappears.
## Your configuration (mandatory)
PyMuPDF 1.18.15: Python bindings for the MuPDF 1.18.0 library.
Version date: 2021-07-10 00:00:01.
Built for Python 3.8 on win32 (64-bit).

## Additional context (optional)
Here are the XPS files and converted PDF files I used for testing.
[xpstest.pdf](https://github.com/pymupdf/PyMuPDF/files/6875199/xpstest.pdf)
[xpstest.zip](https://github.com/pymupdf/PyMuPDF/files/6875201/xpstest.zip)

Thank you very much.

**Comment by JorjMcKie** [comment]
Thank you for the well-prepared issue.
Unfortunately, this is no bug inside PyMuPDF. The PyMuPDF code uses directly MuPDF to do the conversion.
I was able to reproduce the error via CLI command ``mutool convert -o xpstest.pdf xpstest.xps``.

All you can do is submit this bug to https://bugs.ghostscript.com/enter_bug.cgi and attach the XPS file and the above conversion command alongside a short description.

**Comment by JorjMcKie** [comment]
I am moving this issue to **_Discussions_**, alongside other upstream issues.

**Reply by gpjupc888** [reply]
Thank you.
